tutorials/dataframe/df106_HiggsToFourLeptons.py

```diff
@@ -105,36 +105,21 @@
     return (p1 + p2 + p3 + p4).M() / 1000;
 }
 """)
+df = df.Define("m4l", "ComputeInvariantMass(goodlep_pt, goodlep_eta, goodlep_phi, goodlep_E)")
+
+# Book histograms for the four different samples: data, higgs, zz and other (this is specific to this particular analysis)
+histos = [] 
+for sample_category in ["data", "higgs", "zz", "other"]:
+    histos.append(df.Filter(f"sample_category == \"{sample_category}\"").Histo1D(ROOT.RDF.TH1DModel(f"\"{sample_category}\"", "m4l", 24, 80, 170), "m4l", "weight"))
```

Comment:
I don't think the title of the `TH1DModel` needs the quotes right?

```suggestion
    histos.append(df.Filter(f"sample_category == \"{sample_category}\"").Histo1D(ROOT.RDF.TH1DModel(f"{sample_category}", "m4l", 24, 80, 170), "m4l", "weight"))
```